src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -66,6 +66,8 @@ ArcGisMapServerItemViewModel.prototype._enableInCesium = function() {
 
     var scene = this.application.cesium.scene;
 
+    this.legendUrl = this.url + '/legend';
```

Comment:
I'm sorry, we must be talking about two different things, because I have no idea what you mean. :)